core/plugins/org.polarsys.capella.core.compare/src/org/polarsys/capella/core/compare/CapellaComparisonMethod.java

```diff
@@ -111,9 +112,8 @@ public CapellaComparisonMethod(IModelScopeDefinition leftScopeDef,
    */
   @Override
   protected EditingDomain createEditingDomain() {
-    SemanticEditingDomainFactory factory = new SemanticEditingDomainFactory();
-    EditingDomain result = factory.createEditingDomain();
-    return result;
+    ExecutionManager manager = ExecutionManagerRegistry.getInstance().addNewManager();
```

Comment:
This is now consistent with https://github.com/eclipse/capella/blob/409a90c4a2ed6c9c355badee0d068c32ec47f195/core/plugins/org.polarsys.capella.core.compare/src/org/polarsys/capella/core/compare/CapellaScopeFactory.java#L77